package.json

```diff
@@ -25,14 +26,14 @@
     "node": ">0.10.0"
   },
   "dependencies": {
+    "coffee-script": "^1.12.7",
     "bunyan": "^1.0.1",
     "concat-stream": "^1.4.6",
     "iconv-lite": "^0.4.4",
     "minimist": "^1.1.0",
-    "printf": "0.2.x"
+    "printf": "^0.6.1"
   },
   "devDependencies": {
-    "coffee-script": "^1.8.0",
     "grunt": "^0.4.5",
```

Comment:
I did notice that, but those commits no longer even build. They fail with
```
~/projects/node-dicom$ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: dicom@0.4.4
npm ERR! Found: grunt@1.3.0
npm ERR! node_modules/grunt
npm ERR!   grunt@"1.3.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer grunt@"~0.4.0" from grunt-contrib-nodeunit@0.4.1
```

I went further and tried to update `grunt-contrib-nodeunit` and `grunt-contrib-coffee` to resolve that, but they are using coffeescript 2.x under the hood and the tests start failing with these errors
```
node-dicom/src/decoder.coffee:33:13: error: Can't reference 'this' before calling super in derived class constructors if not (this instanceof Decoder)
```

Those errors require us to update the src code to be compatible with coffeescript 2 which didn't turn out to be trivial.

Reply:
I did a little more investigation and here is my understanding of what happened:
- We don't have package-lock.json committed in this repo, the peer deps updated, and we can no longer npm install on the `mica-13552-bad-vr` or `easy-npm-install` branches from scratch.
- Middle and dicomweb still can install the git revisions though, I assume because they have package-lock.json with frozen peer deps.

Fresh `npm install`
```bash
git checkout mica-13552-bad-vr
rm -rf package-lock.json node_modules/
npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: dicom@0.4.4
npm ERR! Found: grunt@1.3.0
npm ERR! node_modules/grunt
npm ERR!   grunt@"1.3.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer grunt@"~0.4.0" from grunt-contrib-nodeunit@0.4.1
```

Fresh `npm install --legacy-peer-deps`
```bash
git checkout mica-13552-bad-vr
npm install --legacy-peer-deps
...
> dicom@0.4.4 postinstall
> coffee --compile --output lib/ src/

/Users/nicholas.svarich/projects/node-dicom/src/decoder.coffee:33:13: error: Can't reference 'this' before calling super in derived class constructors
    if not (this instanceof Decoder)
            ^^^^
npm ERR! code 1
```